extensions/wikibase/tests/src/org/openrefine/wikibase/qa/scrutinizers/FileNameScrutinizerTest.java

```diff
@@ -77,7 +137,7 @@ public void testInvalidCharactersInFilenameVerticalBar() throws IOException, Med
         assertWarningsRaised(FileNameScrutinizer.invalidCharactersInFileNameType);
     }
 
-    @Test
+    @Test(enabled = false) // HTML escaped entities should be allowed
```

Comment:
Same here - I just tried uploading something with this filename and it fails with the error:
`This title is invalid. Make sure to remove characters like square brackets, colons, slashes, comparison operators, pipes and curly brackets.`